doc/manual/src/command-ref/env-common.md

```diff
@@ -71,9 +71,12 @@ Most Nix commands interpret the following environment variables:
     Settings are separated by the newline character.
 
   - <span id="env-NIX_USER_CONF_FILES">[`NIX_USER_CONF_FILES`](#env-NIX_USER_CONF_FILES)</span>\
-    Overrides the location of the user Nix configuration files to load
-    from (defaults to the XDG spec locations). The variable is treated
-    as a list separated by the `:` token.
+    Overrides the location of the Nix user configuration files to load from.
+
+    The default are the locations according to the [XDG Base Directory Specification].
+    See the [XDG Base Directories](#xdg-base-directories) sub-section for details.
```

Comment:
I see you are just clarifying the existing text, but at some point (future PR is fine!) we should probably be more precise.

the default is https://github.com/NixOS/nix/blob/f8a6a9e47314acebea2d72c0ec195360eb58bbec/src/libstore/globals.cc#LL106C33-L106C52 using a `getConfigDirs`

`getConfigDirs` is https://github.com/NixOS/nix/blob/f8a6a9e47314acebea2d72c0ec195360eb58bbec/src/libutil/util.cc#L604 and also uses an undocumented `XDG_CONFIG_DIRS`.

We should document this (XDG config dirs) below and and in the API docs, and then refer to that here rather than just saying "according to the XDG Base Directory Specification" which doesn't make clear which variables are being used.

Reply:
Could you please open an issue for that so we don't forget?

Reply:
Done in #8343